src/SFA.DAS.Roatp.Functions/ApplicationExtract.cs

```diff
@@ -219,5 +223,81 @@ public async Task EnqueueApplyFilesForExtract(IAsyncCollector<ApplyFileExtractRe
                 await applyFileExtractQueue.AddAsync(new ApplyFileExtractRequest(submittedApplicationAnswer));
             }
         }
+
+        private List<OrganisationManagement> ExtractOrganisationManagementAnswers(Guid applicationId, List<SubmittedApplicationAnswer> answers)
+        {
+            var organisationManagementAnswers = new List<OrganisationManagement>();
+
+            var application = _applyDataContext.Apply.Where(app => app.ApplicationId == applicationId).FirstOrDefault();
+            var submittedAnswersOrganisationManagement = answers.Where(answer => answer.SequenceNumber == DeliveringApprenticeshipTraining && answer.SectionNumber == ManagementHierarchy).GroupBy(a => a.RowNumber).ToList();
+
+            foreach (var managementHierarchyPerson in submittedAnswersOrganisationManagement)
+            {
+                var organisationManagement = new OrganisationManagement
+                {
+                    OrganisationId = application.OrganisationId,
+                };
+                foreach (var personDetails in managementHierarchyPerson)
+                {
+                    switch (personDetails.ColumnHeading)
+                    {
+                        case "First Name":
+                            organisationManagement.FirstName = personDetails.Answer;
+                            break;
+                        case "Last Name":
+                            organisationManagement.LastName = personDetails.Answer;
+                            break;
+                        case "Job role":
+                            organisationManagement.JobRole = personDetails.Answer;
+                            break;
+                        case "Years in role":
+                            organisationManagement.TimeInRoleMonths += int.Parse(personDetails.Answer) * MonthsInYear; 
```

Comment:
I am presuming years cannot be blank as otherwise int.Parse will raise error.

Reply:
yeah, I checked the AddManagementHierarchy validator which does similar eparsing (but inexplicably for a long, not an int!), so it will never be blank.  Other code checks for months to be between 1-11, and years between 0 and 99